server/database/init/01-create.sql

```diff
@@ -0,0 +1,18 @@
+set names 'utf8';
```

Comment:
Is this file auto generated? It may be best to add this to the .gitignore

Reply:
The postgres Docker container has a mechanism for [initialization scripts](https://github.com/docker-library/docs/blob/master/postgres/README.md#initialization-scripts), so I generate the schema manually and put it here. It probably *should* be automated or at least documented better